storage/badger/dkg_keys.go

```diff
@@ -27,15 +27,15 @@ func NewDKGKeys(collector module.CacheMetrics, db *badger.DB) (*DKGKeys, error)
 
 	store := func(key interface{}, val interface{}) func(*transaction.Tx) error {
```

Comment:
We could change the name of this storage interface to `BeaconKeys` to match the other name changes